sdks/python/scripts/run_integration_test.sh

```diff
@@ -198,6 +198,7 @@ if [[ -z $PIPELINE_OPTS ]]; then
   # See: https://github.com/hamcrest/PyHamcrest/issues/131.
   echo "pyhamcrest!=1.10.0,<2.0.0" > postcommit_requirements.txt
   echo "mock<3.0.0" >> postcommit_requirements.txt
+  echo "parameterized>=0.7.1,<0.8.0" >> postcommit_requirements.txt
```

Comment:
This was added because the `bigquery_test` module imports `_ELEMENTS` from the `bigquery_file_loads_test` module, which uses `parameterized`. Should we refactor `_ELEMENTS` into its own module?